Sources/Classes/Headers/Public/RSDBPersistentManager.h

```diff
@@ -36,6 +36,7 @@ NS_ASSUME_NONNULL_BEGIN
 -(void) clearProcessedEventsFromDB;
 - (int) getDBRecordCountForMode:(MODES) mode;
 -(void) flushEventsFromDB;
+-(void) markUnProcessedDeviceModeEventsStatusesAsDeviceModeProcessingDone;
```

Comment:
A long method name is bad practice. Keep it as simple as possible. Put a comment if needed.